src/engine/client/sound.cpp

```diff
@@ -63,22 +64,17 @@ static int m_CenterY = 0;
 static float m_MaxDistance = 1500.0f;
 
 static int m_MixingRate = 48000;
-static volatile int m_SoundVolume = 100;
+static int m_SoundVolume = 100;
 
 static int m_NextVoice = 0;
 static int *m_pMixBuffer = 0;	// buffer only used by the thread callback function
 static unsigned m_MaxFrames = 0;
 
 static IOHANDLE s_File;
 
-// TODO: there should be a faster way todo this
 static short Int2Short(int i)
```

Comment:
Should be inline instead of static (thou the compiler is probably able to figure this out anyway).